docs/calico-with-docker/docker-network-plugin/README.md

```diff
@@ -159,6 +159,10 @@ To create the networks, run:
 
 #### c. Networking using default IPAM in a non-cloud environment
 
+With the release of Docker 1.10, support has been added to allow users to
```

Comment:
You can use Calico IPAM.  As with default IPAM you must specify a subnet on the network.  With Calico IPAM the subnet must equal one of the configured IP Pools.

I would add a separate section about specifying a separate IP address rather than builking out each of these separate sections.